core/amber/src/main/python/core/runnables/test_console_message.py

```diff
@@ -28,17 +27,15 @@ def timed_buffer(self):
 
     @pytest.fixture
     def console_message(self):
-        return PythonConsoleMessageV2(
-            ConsoleMessage(
-                worker_id="0",
-                timestamp=current_time_in_local_timezone(),
-                # this will produce error if betterproto is set to 2.0.0b7
-                # timestamp=datetime.datetime.now(),
-                msg_type=ConsoleMessageType.PRINT,
-                source="pytest",
-                title="Test Message",
-                message="Test Message",
-            )
+        return ConsoleMessage(
+            worker_id="0",
+            timestamp=current_time_in_local_timezone(),
+            # this will produce error if betterproto is set to 2.0.0b7
+            # timestamp=datetime.datetime.now(),
```

Comment:
is this comment still valid? we have updated to b7 now.

Reply:
it does not produce error.